app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -651,7 +651,11 @@ public void find_new_raw_curve() {
             BgReading latest_entry = BgReading.lastNoSenssor();
             ra = 0;
             rb = 0;
-            rc = latest_entry.age_adjusted_raw_value;
+            if (latest_entry != null) {
```

Comment:
Here is a call stack of the issue:
E/AndroidRuntime(17595): FATAL EXCEPTION: Thread-31340
E/AndroidRuntime(17595): java.lang.NullPointerException
E/AndroidRuntime(17595):        at com.eveningoutpost.dexdrip.Models.BgReading.find_new_raw_curve(BgReading.java:655)
E/AndroidRuntime(17595):        at com.eveningoutpost.dexdrip.Models.BgReading.perform_calculations(BgReading.java:522)
E/AndroidRuntime(17595):        at com.eveningoutpost.dexdrip.Models.BgReading.create(BgReading.java:296)
E/AndroidRuntime(17595):        at com.eveningoutpost.dexdrip.Services.WixelReader.setSerialDataToTransmitterRawData(WixelReader.java:408)
E/AndroidRuntime(17595):        at com.eveningoutpost.dexdrip.Services.WixelReader.run(WixelReader.java:378)
I/ActivityManager( 2343): Notify an ApplicationCrash

I believe that the problem comes from the fact that if we have a sensor, but no calibration yet, we do not have a calculated_value.
Since lastNoSenssor should checks for calculated_value != 0 it will return null for the first bg reading.
Do you think that we should do another fix for that?
(by the way, I also wonder why I only get this for syntactic testing, and not in real life...)